CONTRIBUTING.md

```diff
@@ -85,28 +44,36 @@ good reason for it, but "I prefer it this way" isn't a strong enough reason -
 there is real value to a standard style, because diverging from it sends a
 message that the code in question is special and care should be taken.
 
+- **Code should be written with python 3 syntax** - This means "new" exception
```

Comment:
This is a little confusing now being moved under "Guidelines" because it will cause test failures and build breakage